Data/Repositories/RecruitmentRepository.cs

```diff
@@ -27,27 +27,13 @@ public IQueryable<Recruitment> GetAllRecruitments()
             return result;
         }
 
-        public int AddRecruitment(Recruitment recruitment)
+        public void AddRecruitment(Recruitment recruitment)
         {
-            try
-            {
                 AddAndSaveChanges(recruitment);
-            }catch(Exception ex)
-            {
-                return -1;
-            }
-            return 1;
         }
-        public int UpdateRecruitment(Recruitment recruitment)
+        public void UpdateRecruitment(Recruitment recruitment)
```

Comment:
Why you need this method?

Reply:
Same as above

Reply:
But you already have method to Add or Update entity in this repository

Reply:
The way I see it I don't. I only have those. Inside of them I am using the mehods from the BaseRepository. Is that what you mean?

Reply:
Yes, you can use methods from BaseRepository